src/Chainweb/MerkleLogHash.hs

```diff
@@ -136,7 +136,7 @@ randomMerkleLogHash :: MerkleHashAlgorithm a => MonadIO m => m (MerkleLogHash a)
 randomMerkleLogHash = unsafeMerkleLogHash <$> randomByteString merkleLogHashBytesCount
 
 merkleLogHashToText :: MerkleHashAlgorithm a => MerkleLogHash a -> T.Text
-merkleLogHashToText = encodeB64UrlNoPaddingText . runPutS . encodeMerkleLogHash
+merkleLogHashToText (MerkleLogHash bytes) = encodeB64UrlNoPaddingText (encodeMerkleRoot bytes)
```

Comment:
This is equivalent?

Reply:
The second is potentially more efficient (since it avoid a trivial roundtrip through a builder). However, the original code is more idiomatic and friendlier to maintain, because all other encodings are derived from `encodeMerkleHashLog`. It is also unrelated to this PR.

I'll to revert this changes.